docs/reference/release-notes/860.md

```diff
@@ -19,6 +19,71 @@ These release notes identify the new features included in 8.6, including [alpha
 | ---------------------- | ---------------------------- | ------------ | ------------ | --------------------------------------------------------------------------------------- |
 | 8 October 2024         | 14 April 2026                |              |              | [Self-Managed update guide](self-managed/operational-guides/update-guide/850-to-860.md) |
 
+### Migrate process management endpoints to Camunda 8 REST API
+
+<!-- https://github.com/camunda/product-hub/issues/2040 -->
+
+We're thrilled to announce the integration of Zeebe endpoints into Camunda 8 REST API, aimed at enhancing your experience in building process applications. This will ease the installation and onboarding to Camunda 8 as gRPC won't be required anymore. With the introduction of Zeebe endpoints in REST API, we're eliminating the need for gRPC, reducing friction in adoption. Developers can now seamlessly transition from gRPC to REST with the assurance of having the same endpoints, thereby saving time and effort. Use officially supported Camunda SDKs to have an easy migration journey.
+
+### Investigate and resolve incidents with execution listeners
+
+<!-- https://github.com/camunda/product-hub/issues/2103 -->
+
+Users can now easily check details of active and completed execution listeners for process activities and process instances. Operations engineers can efficiently resolve execution listener incidents. This enhancement streamlines operations and ensures smoother handling of incidents, ultimately improving time to unblock process execution.
+
+### Deprecate zbctl and GO client
```

Comment:
@aleksander-dytko should we link to https://camunda.com/blog/2024/09/deprecating-zbctl-and-go-clients/?